openstudiocore/src/model/CMakeLists.txt

```diff
@@ -1603,28 +1606,43 @@ set(${target_name}_swig_src
 
 qt5_wrap_cpp_minimally(${target_name}_moc_src ${${target_name}_moc})
 
-set(${target_name}_qrc
-  Model.qrc
+set(${target_name}_depends
+  openstudio_utilities
 )
 
-# generate rules for building source files from the resources
-# DLM: TODO, replace with embed_files
-qt5_add_resources(${target_name}_qrcs ${${target_name}_qrc})
+# Embed resource files
+# Glob all files inside model/Resources
+file(GLOB_RECURSE embedded_files "${CMAKE_CURRENT_SOURCE_DIR}/Resources/**/*.*")
```

Comment:
@jmarrec "resources" is lower case right?

Reply:
I renamed it to "Resources" to match what's used in other places (eg: src/energyplus/Resources, src/openstudio_app/Resources), so that all "Resources" files currently end up in the same "embedded" folder.

Reply:
Ok, just as an FYI people may need to manually correct this as I don't think git will re-case directories